heralding/capabilities/ssh.py

```diff
@@ -40,6 +41,10 @@ def connection_made(self, conn):
         logger.debug('SSH connection received from %s.' % conn.get_extra_info('peername')[0])
 
     def connection_lost(self, exc):
+        # log auxiliary info
+        aux_data = self.get_auxiliary_info()
+        self.session.add_auxiliary_info(aux_data)
```

Comment:
I would rename `get_auxiliary_info` into `log_auxiliary_info` and  put `self.session.add_auxiliary_info(aux_data)` inside. Thoughts? :)